packages/editor/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 - Removed `jQuery` dependency
 
+### Deprecations
+
+- The `getAutosave`, `getAutosaveAttribute`, and `hasAutosave` selectors are deprecated. Equivalent selectors have been added to the `@wordpress/core-data` package.
+- The `resetAutosave` action creator is deprecated. An equivalent action creator `receiveAutosave` has been added to the `@wordpress/core-data` package.
```

Comment:
I don't know that "action creator" as in the Redux lexicon quite fits for how we expose these things, since they're already prebound to dispatch. As far as public documentation goes, I find it's okay to refer to them as "actions", or perhaps "action dispatchers".

Reply:
Have updated this to just 'actions'.